train_segmentation_model_prediction_thresholds.py

```diff
@@ -161,5 +180,20 @@ def train_segmentation_model_prediction_thresholds(gcp_bucket, dataset_directory
         type=float,
         default=1.0,
         help='Accelerate optimization via using subset of dataset.')
+    argparser.add_argument(
+        '--python-random-global-seed',
```

Comment:
this is pretty nitpicky but imo this may be confusing to future users to call it the `python-random-global-seed` -- I'd suggest `random-module-global-seed` (eg as its referred to here: https://docs.python.org/3/library/random.html )

Reply:
okay will do!